flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/ExecutionGraphPartitionReleaseTest.java

```diff
@@ -179,10 +176,10 @@ private SchedulerBase createScheduler(
 			final JobVertex... vertices) throws Exception {
 
 		final JobGraph jobGraph = new JobGraph(new JobID(), "test job", vertices);
-		final SlotProvider slotProvider = new TestingSlotProvider(
-			ignored -> CompletableFuture.completedFuture(new TestingLogicalSlotBuilder().createTestingLogicalSlot()));
-		final SchedulerBase scheduler = SchedulerTestingUtils
-			.newSchedulerBuilderWithDefaultSlotAllocator(jobGraph, slotProvider, Time.seconds(10))
+		final SchedulerBase scheduler = SchedulerTestingUtils.newSchedulerBuilder(jobGraph)
+			.setExecutionSlotAllocatorFactory(SchedulerTestingUtils.slotSharingExecutionSlotAllocatorFactoryBuilder()
```

Comment:
Can't this be the default? Or is it the custom allocation timeout we are setting here?

Reply:
Yes, I want to consider the custom allocation time here.

Reply:
Fair enough. I removed it. Initially, I just thought of keeping the test code as is.